templates/code/searchform.tmpl

```diff
@@ -0,0 +1,14 @@
+<form class="ui form ignore-dirty" style="max-width: 100%">
+	<div class="ui fluid action input">
+		<input name="q" value="{{.Keyword}}"{{if .CodeIndexerUnavailable }} disabled{{end}} placeholder="{{.locale.Tr "explore.search"}}..." autofocus>
```

Comment:
Hmm, is it such a good idea to use `...` instead of `…` (U+2026, &hellip)?
Firstly, it looks at least as good if not even better as you can see.
Secondly, it probably helps with a11y, because I doubt screen readers will pronounce it as `dot dot dot` or similar.

---

I know that is used inside other templates.
If you want, I can also create a separate PR to change all instances.

Reply:
changed but that is coming from other code search forms, fixed also in them